.github/workflows/linux.yml

```diff
@@ -87,6 +87,36 @@ jobs:
             -DAMReX_PARTICLES=ON
         make -j 2
 
+  test_sensei:
+    name: SENSEI Adaptor [test]
+    runs-on: ubuntu-18.04
+    env:
+      CXX: clang++
+      CC: clang
+      CXXFLAGS: "-Werror -Wshadow -Woverloaded-virtual -Wunreachable-code -fno-operator-names"
```

Comment:
@kwryankrattiger I don't think -Werror should be on, this would potentially cause the build to fail for reasons unrelated to the SENSEI code.

Reply:
Oh, we add this to avoid that various code sections introduce warnings with new PRs :)

Reply:
Ok, sounds good then!